.github/workflows/setup-fortran.yml

```
@@ -20,7 +20,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         toolchain:
-          - {compiler: gcc, version: 14}
+          - {compiler: gcc, version: 14, FFLAGS: "-Werror=unused"}
```

I think that if we set `FPM_FFLAGS` then we don't need to set it below. I suppose we could set both here?

```suggestion
          - {compiler: gcc, version: 14, FPM_FFLAGS: "-Werror=unused"}
```